contracts/test/Summa.ts

```diff
@@ -255,113 +242,85 @@ describe("Summa Contract", () => {
-              assetName: "ETH",
-              amount: BigNumber.from(556863),
-            },
-            {
-              chain: "ETH",
-              assetName: "USDT",
-              amount: BigNumber.from(556863),
-            },
-          ],
-          proof,
+          BigNumber.from(1000000000),
+          {
+            chain: "ETH",
+            assetName: "ETH",
+            amount: BigNumber.from(556863),
+          },
           BigNumber.from(1693559255)
         )
       ).to.be.revertedWith("Ownable: caller is not the owner");
     });
 
     it("should not verify the proof of solvency if the CEX hasn't proven the address ownership", async () => {
       await expect(
-        submitProofOfSolvency(summa, mstRoot, proof)
+        submitCommitment(summa, mstRoot, rootSum)
       ).to.be.revertedWith(
         "The CEX has not submitted any address ownership proofs"
       );
     });
 
-    it("should revert with invalid MST root", async () => {
-      mstRoot = BigNumber.from(0);
+    it("should revert with invalid root sum", async () => {
+      rootSum = BigNumber.from(0);
 
       await summa.submitProofOfAddressOwnership(ownedAddresses);
 
       await expect(
-        submitProofOfSolvency(summa, mstRoot, proof)
-      ).to.be.revertedWith("Invalid ZK proof");
+        submitCommitment(summa, mstRoot, rootSum)
+      ).to.be.revertedWith("Root sum should be greater than zero");
     });
 
     it("should revert with invalid assets", async () => {
       await summa.submitProofOfAddressOwnership(ownedAddresses);
 
       await expect(
-        submitProofOfSolvency(summa, mstRoot, proof, [
-          {
-            chain: "",
-            assetName: "ETH",
-            amount: BigNumber.from(556863),
-          },
-        ])
+        submitCommitment(summa, mstRoot, rootSum, {
+          chain: "",
+          assetName: "ETH",
+          amount: BigNumber.from(556863),
+        })
       ).to.be.revertedWith("Invalid asset");
 
       await expect(
-        submitProofOfSolvency(summa, mstRoot, proof, [
-          {
-            chain: "ETH",
-            assetName: "",
-            amount: BigNumber.from(556863),
-          },
-        ])
+        submitCommitment(summa, mstRoot, rootSum, {
+          chain: "ETH",
+          assetName: "",
+          amount: BigNumber.from(556863),
+        })
       ).to.be.revertedWith("Invalid asset");
     });
-
-    it("should revert with invalid proof", async () => {
-      await summa.submitProofOfAddressOwnership(ownedAddresses);
-
-      proof = proof.replace("1", "2");
-
-      await expect(
-        submitProofOfSolvency(summa, mstRoot, proof)
-      ).to.be.revertedWith("Invalid ZK proof");
-
-      proof = "0x000000";
-
-      await expect(
-        submitProofOfSolvency(summa, mstRoot, proof)
-      ).to.be.revertedWithoutReason();
-    });
   });
 
   describe("verify proof of inclusion", () => {
     let mstRoot: BigNumber;
+    let rootSum: BigNumber;
```

Comment:
I cannot comment on the right line but `solvencyProof` should no longer exist here